apps/web/cypress/support/commands.ts

```diff
@@ -17,10 +17,6 @@ Cypress.Commands.add('clickWorkflowNode', (selector: string, last?: boolean) =>
   return cy.getByTestId(selector).click({ force: true });
 });
 
-Cypress.Commands.add('seed', () => {
-  return cy.request('POST', `${Cypress.env('apiUrl')}/v1/testing/seed`, {});
-});
```

Comment:
Using this custom command instead the task allowed that my incorrect local setup was creeping into the tests. And as I had the local environment defined and being used for the test set up of the backend I was having this call to connect to `novu-db` database instead of `novu-test` that is the one called by the tasks defined for Cypress testing.

Reply:
Ohhh now that makes sense. Good job on finding the root cause. I'm totally ok with using the task instead of a direct call. This could also create a helper command to work with auto completion:

```
Cypress.Commands.add('seed', () => {
  return cy.task('seedDatabase')
});
```

But in terms of the actual env behaviour of this, looks like the `cy.task('seedDatabase')` command is not really seeding the database with a test user, which it was only using in the auth.spec.ts, so in case if it's not needed anymore here so we can definitely remove it from here.